src/EventStore.Core/ClusterVNode.cs

```diff
@@ -1752,7 +1752,7 @@ public class ClusterVNode<TStreamId> :
 				}
 			}
 
-			var chainStatus = CertificateUtils.BuildChain(certificate, intermediates, trustedRootCertsSelector());
+			var chainStatus = CertificateUtils.BuildChain(certificate, intermediates, trustedRootCertsSelector(), out _);
```

Comment:
The `chainStatusInformation` returned from this call to `BuildChain` should also be captured and logged to give more detailed information about why other nodes' certificates are invalid